packages/subiquity_client/test/subiquity_client_test.dart

```diff
@@ -19,11 +19,57 @@ void main() {
     expect(await isOpen, isTrue);
   });
 
+  group('subiquity process', () {
+    test('set additional environment before starting the process', () async {
+      const foo = '42';
+      final env = {'TEST_VAR': foo};
+      final process = SubiquityProcess(
+        'bash',
+        ['-c', 'exit \$TEST_VAR'],
+      );
+      await process.start(additionalEnv: env);
+      expect(await process.exitCode, int.parse(foo));
+      await process.stop();
```

Comment:
```suggestion
      await process.start(additionalEnv: env);
      addTearDown(process.stop);
      expect(await process.exitCode, int.parse(foo));
```
ensures clean up even if expect() may fail